pkg/cloud/cloud_test.go

```diff
@@ -1,6 +1,7 @@
 package cloud
 
 import (
+	"github.com/openshift/cluster-cloud-controller-manager-operator/pkg/cloud/alibaba"
```

Comment:
Please move this import into the group with the rest of the cloud providers, it should come before the AWS import on line 9